src/components/Renderer/index.tsx

```diff
@@ -155,21 +185,11 @@ const RendererBase = ({
 	});
 	const processedValue = getValue(value, schema, processedUiSchema);
```

Comment:
I guess we now should be passing the plain `value` to the `Widget`, since the `Widget` is also doing the processing now, right?